tasks/test.py

```diff
@@ -2,12 +2,20 @@
 
 from .util import PROJECT_DIR, print_and_run
 
+# NOTE: running "mypy" instead of "mypy --no-incremental" gave hundreds of
+#       errors.  SS 7/8/2024
+#  Per:
+#    https://stackoverflow.com/questions/52189217/use-mypy-with-ruamel-yaml
+#  ...it appears this is a known issue.  See:
+#    https://github.com/python/mypy/issues/7276
+#    https://sourceforge.net/p/ruamel-yaml/tickets/328/
+
```

Comment:
I'd like to remove the `--no-incremental` bit if we can, I may look into this a bit. This same change was applied on the `seaice_ecdr` repo and undoing it was OK for me. Without the `--no-incremental` flag, it takes quite a bit longer to run the typechecker.